ucm2/sof-soundwire/RT711.conf

```diff
@@ -0,0 +1,44 @@
+# Use case Configuration for sof-soundwire card
+
+If.RT711 {
+	Condition {
+		Type String
+		Haystack "${CardComponents}"
+        	Needle "hs:rt711" 
+	}
+	True {
+
+		SectionDevice."Headphone" {
+			Comment	"Headphone"
+
+			EnableSequence [
+				cset "name='rt711 DAC Surr Playback Volume' 87"
+			]
+
+			Value {
+			      PlaybackPriority 200
+			      PlaybackPCM "hw:${CardId},0"
+			      PlaybackChannels "2"
+			      JackControl "Headphone Jack"
+			}
+		}
+
+		SectionDevice."Microphone" {
+			Comment "Headset Mic"
+
+			EnableSequence [
+				cset "name='rt711 ADC 23 Mux' 'MIC2'"
+				cset "name='rt711 ADC 08 Capture Volume' 63"
+				cset "name='rt711 ADC 08 Capture Switch' 1"
+				cset "name='rt711 AMIC Volume' 1"
+			]
+
```

Comment:
missed DisableSequence?

Reply:
those where the settings we've used so far, I don't know if there is a need to turn something off.
Likewise from a hardware perspective, there is no conflict between headphone and speaker, they can work at the same time, but that may get in the way of user-experience

Reply:
OK. Mostly if we switch on something in enablesequence, we will switch off it in disablesequence unless it impacts on other streams. However, even we don't switch off it, it still works well because pulseaudio will switch stream automatically.

Reply:
how about to unify this UCM setting as other codecs ?
````			
DisableSequence [
				cset "name='rt711 ADC 08 Capture Switch' 0"
	        ]
````